mlrun/datastore/targets.py

```diff
@@ -121,12 +123,20 @@ def validate_target_paths_for_engine(targets, engine, source):
                     f"spark CSV/Parquet targets must be directories, got path:'{target.path}'"
                 )
             elif engine == "pandas":
-                if source.attributes.get("chunksize"):
+                if isinstance(source, DataSource) and source.attributes.get(
+                    "chunksize"
+                ):
                     if is_single_file:
                         raise mlrun.errors.MLRunInvalidArgumentError(
                             "pandas CSV/Parquet targets must be a directory "
                             f"for a chunked source, got path:'{target.path}'"
                         )
+                elif target.kind == TargetTypes.parquet and target.partitioned:
+                    if is_single_file:
```

Comment:
```suggestion
                elif target.kind == TargetTypes.parquet and target.partitioned and is_single_file:
```

Reply:
in case of parquet and partitioned - if not single file then it is valid (shouldn't go to next validation)

Reply:
But the next validation is `not is_single_file` 😄. Anyway it's not really important.